RealmSwift-swift2.0/Tests/MigrationTests.swift

```diff
@@ -146,6 +149,7 @@ class MigrationTests: TestCase {
         }
     }
 
+    // swiftlint:disable function_body_length
```

Comment:
I think we certainly *should* address these at some point. Refactoring long tests to test more concisely is a nice goal.